src/pug/includes/head.pug

```diff
@@ -7,6 +7,14 @@ meta(http-equiv="X-UA-Compatible" content="IE=edge")
 
 meta(name="description" content="swag opportunities for developers")
 
+meta(property="og:title" content="devSwag")
+meta(property="og:description" content="Swag opportunities for developers")
+meta(property="og:site_name" content="devSwag")
+meta(property="og:type" content="website")
+meta(property="og:url" content="https://devswag.io")
+meta(property="og:image" content="/assets/img/devswag-card.png")
```

Comment:
Did you make this?

Reply:
Yes, I did this to create a preview photo on cards

Reply:
Alright, just checking! I'll let the other maintainers review it, but I'd personally  prefer a bit more whitespace after the divider

Reply:
@aslafy-z and @swapagarwal what do you think about the photo card

Reply:
It looks good! But I'm okay with @vikaspotluri123, I think it needs some more spacing.

Reply:
Just waiting on this 😄

Reply:
ping @sarscode

Reply:
ping @sarscodes, any chance you get some time to edit this photo card? It would be awesome!